tests/unit/engines/autoencoders/inference/test_embedding_component.py

```diff
@@ -0,0 +1,21 @@
+# pylint: disable-all
+import pytest
+
+from recommender.engines.autoencoders.inference.embedding_component import (
+    EmbeddingComponent,
+)
+from recommender.engines.autoencoders.training.model_training_step import (
+    AEModelTrainingStep,
+)
+
+
+def test_embedding_component(simulate_data_preparation_step, ae_pipeline_config):
+    ae_model_train_step = AEModelTrainingStep(ae_pipeline_config)
+
+    data_prep_step, _ = simulate_data_preparation_step
+    _, _ = ae_model_train_step(data_prep_step)
+
+    # Save Embedders
+    ae_model_train_step.save()
+
+    EmbeddingComponent()()
```

Comment:
We should add some assertions here, for example if the dense tensors in the DB are of a certain shape and maybe some more